_episodes/02-coords.md

```diff
@@ -596,8 +596,8 @@ This is something we will need to do multiple times. We will write a function to
 The following function combines these steps. 
 
 ~~~
-def skycoord_to_string(skycoord):
-    """Convert SkyCoord to string."""
+def polygon_skycoord_to_gaia_string(skycoord):
```

Comment:
```suggestion
def skycoord_to_string(skycoord):
```

Reply:
I recommend keeping the shorter function name, but expanding the function description.